locales/en-US/strings.properties

```diff
@@ -80,7 +80,7 @@ prefs_restore_defaults_button=Restore Defaults
 prefs_section_rows_option={num} row;{num} rows
 prefs_search_header=Web Search
 prefs_topsites_description=The sites you visit most
-prefs_topstories_description=High-quality content you might otherwise miss
+prefs_topstories_description2=Great content from around the web, personalized for you
```

Comment:
So you don't need to add `prefs_topstories_description` to [Bug 1445090](https://bugzilla.mozilla.org/show_bug.cgi?id=1445090) then?

Reply:
Nah.. unless we expect to uplift the about:preferences stuff to beta 60.. Although if we do that, we could uplift strings too ;)